Add-ons/Umbraco-Deploy/Extending/index.md

```diff
@@ -7,361 +7,365 @@ meta.Description: "How to extend Umbraco Deploy to synchronize custom data"
 
 # Extending Umbraco Deploy
 
-Umbraco Deploy supports the deployment of CMS schema information, definitions from the HQ's Forms package, along with managed content and media.  In addition it can be extended by package or custom solution developers to allow the deployment of custom data, such as that stored in your own database tables.
+Umbraco Deploy supports the deployment of CMS schema information and definitions from the HQ's Forms package, along with managed content and media. Additionally, it can be extended by package or custom solution developers. This allows the deployment of custom data, such as that stored in your own database tables.
 
-As a package or solution developer, you can hook into the disk based serialization and deployment - similar to that used for Umbraco Document Types and Data Types.  It's also possible to provide the ability for editors to deploy custom data via the Backoffice - in the same way that Umbraco content and media can be queued for transfer and restored.
+As a package or solution developer, you can hook into the disk-based serialization and deployment. It is similar to that used for Umbraco Document Types and Data Types.  It's also possible to provide the ability for editors to deploy custom data via the Backoffice. In the same way that Umbraco content and media can be queued for transfer and restored.
 
 ## Concepts and Examples
 
 ### Entities
 
-*Entities* are what you may be looking to transfer between two websites using Deploy.  Within Umbraco, they are the Document Types, Data Type, documents etc.  In a custom solution or a package, they may be representations of some other data that's being stored separately from Umbraco content, whilst still managed in the Backoffice using custom trees and editors.
+*Entities* are what you may be looking to transfer between two websites using Deploy. Within Umbraco, they are the Document Types, Data Type, documents etc. In a custom solution or a package, there may be representations of some other data that's being stored separately from Umbraco content. These can still be managed in the Backoffice using custom trees and editors.
 
-For the purposes of subsequent code samples, we'll consider an example entity as a POCO class with a few properties.  Note that the entity has no dependency on Umbraco or Umbraco Deploy; it can be constructed and managed however makes sense for the package or solution.  The only requirement is that it has an ID that will be consistent across the environments (normally a Guid) and a name.
+For the purposes of subsequent code samples, we'll consider an example entity as a POCO class with a few properties.
```

Comment:
Yes, it is an acronym to the term plain old class object (POCO). But I'm not sure, in this case, if we should spell out the term  in the docs 🤔

Reply:
Yup, exactly my thoughts. It doesn't sound right in this case 🙈

Reply:
The defition of POCO in this case is "Plain Old CLR Object" or the one you found - both can be used.
I think we should go for the one you suggested then, as `CLR` is just another acronym, and I think we should try to avoid having acronym-inception in our docs 🙈 

We could rewrite the sentence a bit like this:

```suggestion
For the purposes of subsequent code samples, we'll consider an example entity as a plain old class object (POCO) with a few properties.
```

What do yout think?